lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -215,10 +215,11 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
 
   /// Return false if we fail to create the right DW_OP_LLVM_fragment operand.
   bool handleFragmentDIExpr(const SILDIExprOperand &CurDIExprOp,
-                            SmallVectorImpl<uint64_t> &Operands);
+                            std::pair<unsigned, unsigned> &Fragment);
```

Comment:
What do you think about using an `llvm::DIExpression::FragmentInfo` here, so it's more clear which is which?

Reply:
Good idea. Done.